mdpow/fep.py

```diff
@@ -1366,17 +1372,24 @@ def p_transfer(G1, G2, **kwargs):
 
         # for this version. use the method given instead of the one in the input cfg file
         G.method = kwargs.pop('method', 'MBAR')
```

Comment:
The whole loop logic needs to be fixed.
```python
 for G in (G1, G2):
      G_kwargs = kwargs.copy()
       # then use G_kwargs.pop() etc in this loop.
       ...
```
At the moment, `G2` gets an almost empty `kwargs` and uses defaults for everything.